### ISSUE TITLE
Practical Open Science and digitalization guide for LIKE students missing

### ISSUE THREAD

**Issue by InesWuerth** [LIKE-ITN/FAIR-data-and-Open-Science-principles#10]
I am missing a practical guide of Open Science and Digitalization for the LIKE ESRs. I suggest we include this information in the chapter that is now called "Use cases". @AndyClifton could you rename the chapter into something like "Open Science and Digitalization in LIKE"? then we could start gathering tipps for the students there.

**Comment by nikokaoja** [comment]
@InesWuerth as discussed with @AndyClifton , I had to complete the report by today so Charlotte can submit it to the EC portal. As you noticed, there is a section which is currently called Use Cases, which will act as a practical guide for applying Open Science, FAIR and R5 principles for projects ESRs have.  If you could come up with a bit shorter title than "Open Science and Digitalization for the LIKE ESRs" since that will span over two rows online ?

**Comment by AndyClifton** [comment]
Suggest we consider this a task for [release 1.0](https://github.com/LIKE-ITN/FAIR-data-and-Open-Science-principles/milestone/2).

**Comment by InesWuerth** [comment]
@niva83 good point that it's too long. How about just calling the chapter "Application in LIKE"? I'm happy to discuss content for the release 1.0.

**Comment by nikokaoja** [comment]
@InesWuerth sounds good too me :)
